### PR TITLE
redirect vscode content to code.visualstudio.com

### PR DESCRIPTION
**Pull request by craigshoemaker**
redirects traffic from an outdated article to newer content on code.visualstudio.com

### CODE REVIEW

File: .openpublishing.redirection.json [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
+      "redirect_url": "/javascript/azure",
+      "redirect_document_id": false
+    },
+    {
+      "source_path": "node-sdk-azure-install.md",
+      "redirect_url": "/javascript/node-sdk-azure-get-started",
+      "redirect_document_id": false
+    },
+    {
+      "source_path": "javascript/azure/node-howto-e2e.md",
```
[the remark this answers is not in the document]
Comment (craigshoemaker) [reply .openpublishing.redirection.json]: 👍

**Comment by craigshoemaker** [comment]
@mikekinsman do you know why this PR is stuck in the OpenPublishing.Build validation step?

**Comment by mikekinsman** [comment]
@craigshoemaker  never seen this before.  Checking, and I'll ping some folks if I can't figure out whats what

**Comment by craigshoemaker** [comment]
per @wuhanyumsft this PR has passed validation checks, but the GitHub PR UI is not being updated correctly, so I can safely #sign-off on this PR

**Pull request merged**